File: java/praktik.java
import java.util.Scanner;

public class StrukPembelian {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        String nama;
        double harga;
        int jumlah;

        System.out.print("Masukkan nama barang : ");
        nama = input.nextLine();

        System.out.print("Masukkan harga barang: ");
        harga = input.nextDouble();

        System.out.print("Masukkan jumlah beli : ");
        jumlah = input.nextInt();

        double total = harga * jumlah;
        double diskon;

        if (total > 431000) {
            diskon = total * 0.1; // 10% diskon
        } else {
            diskon = 0;
        }

        double total_bayar = total - diskon;

        System.out.println("\n===== STRUK PEMBELIAN =====");
        System.out.println("Nama Barang   : " + nama);
        System.out.println("Harga Satuan  : " + harga);
        System.out.println("Jumlah Beli   : " + jumlah);
        System.out.println("Total Harga   : " + total);
        System.out.println("Diskon        : " + diskon);
        System.out.println("Total Bayar   : " + total_bayar);
        System.out.println("============================");
    }
}

File: php/praktik.php
<?php

// Input
echo "Masukkan nama barang: ";
$nama = trim(fgets(STDIN));

echo "Masukkan harga barang: ";
$harga = floatval(trim(fgets(STDIN)));

echo "Masukkan jumlah beli: ";
$jumlah = intval(trim(fgets(STDIN)));

// Proses
$total = $harga * $jumlah;

if ($total > 431000) {
    $diskon = $total * 0.1;
} else {
    $diskon = 0;
}

$total_bayar = $total - $diskon;

// Output
echo "\n====== STRUK PEMBELIAN ======\n";
echo "Nama Barang   : $nama\n";
echo "Harga Satuan  : $harga\n";
echo "Jumlah Beli   : $jumlah\n";
echo "Total Harga   : $total\n";
echo "Diskon        : $diskon\n";
echo "Total Bayar   : $total_bayar\n";
echo "==============================\n";

?>
